app/controllers/sensor_readings_controller.rb

```diff
@@ -13,6 +13,24 @@ def index
     @sensor_readings
   end
 
+  def update
+    @sensor_reading = Sensor::Reading.find(sensor_reading_delete_params['sensor_reading_id'])
+    respond_to do |format|
+      if @sensor_reading.update(sensor_reading_params)
+        sensor = @sensor_reading.sensor
+        report = sensor.report
+        format.html { redirect_to report_sensor_path(report, sensor), notice: 'Sensor reading was successfully updated.' }
+        format.json { render :show, status: :accepted, location: report_sensor_reading_url(@report, @sensor_reading) }
+      else
+        format.html do
+          flash[:error] = 'Sensor reading could not updated!'
```

Comment:
There’s a typo + passive 
```suggestion
          flash[:error] = 'There was an error updating the sensor reading!'
```